pyspark_ai/prompt.py

```diff
@@ -119,7 +149,8 @@
 
 SPARK_SQL_PREFIX = """You are an assistant for writing professional Spark SQL queries. 
 Given a question, you need to write a Spark SQL query to answer the question. The result is ALWAYS a Spark SQL query.
-"""
+ALWAYS use the tool similar_value to find the correct filter value format.
```

Comment:
Shall we make it optional? Only when the assistant is not sure about the column value